build.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+cd ./build
+cmake .. -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release
```

Comment:
I dont get why would you need this script :)

if you write conan file correctly you just need to add to your readme

```
mkdir build
cd build
conan install ..
conan build ..
```